tests/openassetio-core-c/private/hostAPI/ManagerTest.cpp

```diff
@@ -117,9 +157,14 @@ SCENARIO("A host calls Manager::identifier") {
     managerAPI::ManagerInterfacePtr mockManagerInterfacePtr =
         openassetio::makeShared<MockManagerInterface>();
     auto& mockManagerInterface = static_cast<MockManagerInterface&>(*mockManagerInterfacePtr);
+    // Create a HostSession with our mock HostInterface
+    managerAPI::HostSessionPtr hostSessionPtr =
+        openassetio::makeShared<openassetio::managerAPI::HostSession>(
+            openassetio::makeShared<managerAPI::Host>(
+                openassetio::makeShared<MockHostInterface>()));
 
     // Create the Manager under test.
-    hostAPI::Manager manager{mockManagerInterfacePtr};
+    hostAPI::Manager manager{mockManagerInterfacePtr, hostSessionPtr};
```

Comment:
[consider][minor] I think this chunk of bootstrapping (from after `GIVEN` to `actualErrorMsg`) is long enough and duplicated enough times to warrant a fixture struct (a bit like `InfoDictionaryFixture`).